src/net.cpp

```diff
@@ -910,8 +910,10 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     bool legacyWhitelisted = false;
     if (NetPermissions::HasFlag(permissionFlags, NetPermissionFlags::PF_ISIMPLICIT)) {
         NetPermissions::ClearFlag(permissionFlags, PF_ISIMPLICIT);
+        // DEFAULT_WHITELISTFORCERELAY = false
         if (gArgs.GetBoolArg("-whitelistforcerelay", false)) NetPermissions::AddFlag(permissionFlags, PF_FORCERELAY);
-        if (gArgs.GetBoolArg("-whitelistrelay", false)) NetPermissions::AddFlag(permissionFlags, PF_RELAY);
+        // DEFAULT_WHITELISTRELAY = true
```

Comment:
What is this?

`DEFAULT_WHITELISTRELAY` has nothing to do with validity of blocks or transactions. Please just move it from validation to net or something